src/System.Management.Automation/FormatAndOutput/common/StringDecorated.cs

```diff
@@ -64,20 +67,18 @@ public StringDecorated(string text)
         /// <returns>Rendered string based on outputRendering.</returns>
         public string ToString(OutputRendering outputRendering)
         {
-            if (!_isDecorated)
+            if (outputRendering == OutputRendering.Host)
             {
-                return _text;
+                throw new ArgumentException(StringDecoratedStrings.RequireExplicitRendering);
```

Comment:
It is not good to throw in _public_ ToString().

Reply:
If you take a closer look, you will find it won't throw in `ToString()`. The exception could happen only when user calling `ToString(OutputRendering.Host)`.